ta_services/keymaster/km.c

```diff
@@ -5,6 +5,27 @@
 
 #include "km.h"
 
+bool version_info_set = false;
+uint32_t boot_os_version = 0;
+uint32_t boot_os_patchlevel = 0;
+
+TEE_Result km_configure(uint32_t os_version, uint32_t os_patchlevel)
+{
+	IMSG("os_version = %u", os_version);
```

Comment:
So we'll log this as if this information is updated regardless of if it's ignored or not?

Reply:
Will add more code to make this clearer.